content/en/account_management/billing/usage_attribution.md

```diff
@@ -1,27 +1,29 @@
 ---
-title: Custom usage reporting
+title: Usage attribution
 kind: documentation
 aliases:
   - /account_management/billing/advanced_usage_reporting/
+  - /account_management/billing/custom_usage_reporitng/ 
 ---
 
 ## Overview
 
-Administrators can access the Custom Reporting tab from the Plan & Usage section in Datadog. The Custom Reporting page provides the following information and functionality:
+Administrators can access the Usage Attribution tab from the Plan & Usage section in Datadog. The Usage Attribution page provides the following information and functionality:
 
 * Lists the existing tag keys that usage is being broken down by and provides the ability to change and add new ones (up to three tag keys).
 * Generates daily .tsv (tab separated values) files for most usage types.
-* Summarizes usage at the end of each month. Surfaces both in the UI and as a .tsv download.
+* Summarizes usage at the end of each month. 
+* Surfaces the data both in the UI and as a .tsv download.
 
-**Note**: The following usage types are not supported in this tool:
+**Note**: The following usage types are not supported in this tool yet. They will be added subsequently:
 
 * Indexed Log Events
 * Ingested Logs
 * Analyzed Spans
 
 ### Getting started
 
-To start receiving daily data, an administrator needs to create a new report through the user interface. 
+In order to start receiving daily data, an administrator will need to create a new report via the user interface. 
```

Comment:
```suggestion
To start receiving daily data, an administrator needs to create a new report with the user interface. 
```
Style guide - no future tense, no use of the word `via` for translation purpose, `in order` is unnecessary